### ISSUE TITLE
quest : Troll Patrol: Whatdya Want, a Medal?   can`t loot medals from dead trolls 

### ISSUE THREAD

**Issue by nephelin** [dalaranwow/dalaran-wow#5170]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Thank you for the report @nephelin,
For future reports please fill the entire template so we get all the needed information.
Issue confirmed on live.
Clicking on npc https://wowgaming.altervista.org/aowow/?npc=28162 does not produce a quest item.

**Comment by TeridanDW** [comment]
The SAI for the creature from TC:
-- Drakkari Corpse SAI
SET @ENTRY := 28162;
UPDATE `creature_template` SET `AIName`="SmartAI" WHERE `entry`=@ENTRY;
DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
(@ENTRY,0,0,1,8,0,100,1,51026,0,0,0,11,61286,2,0,0,0,0,7,0,0,0,0,0,0,0,"Drakkari Corpse - On Spellhit 'Create Drakkari Medallion Cover' - Cast 'World Generic Dismount/Cancel Shapeshift' (No Repeat)"),
(@ENTRY,0,1,2,61,0,100,0,51026,0,0,0,11,50737,2,0,0,0,0,7,0,0,0,0,0,0,0,"Drakkari Corpse - On Spellhit 'Create Drakkari Medallion Cover' - Cast 'Create Drakkari Medallion' (No Repeat)"),
(@ENTRY,0,2,3,61,0,100,0,51026,0,0,0,81,0,0,0,0,0,0,1,0,0,0,0,0,0,0,"Drakkari Corpse - On Spellhit 'Create Drakkari Medallion Cover' - Set Npc Flag  (No Repeat)"),
(@ENTRY,0,3,0,61,0,100,0,51026,0,0,0,41,3000,0,0,0,0,0,1,0,0,0,0,0,0,0,"Drakkari Corpse - On Spellhit 'Create Drakkari Medallion Cover' - Despawn In 3000 ms (No Repeat)"),
(@ENTRY,0,4,0,25,0,100,0,0,0,0,0,81,16777216,0,0,0,0,0,1,0,0,0,0,0,0,0,"Drakkari Corpse - On Reset - Set Npc Flag Spellclick");

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, I did just fix it yesterday.
